backend/authentication/adminPasswordRecover.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+const AWS = require('aws-sdk');
+const db = new AWS.DynamoDB.DocumentClient();
+const SES = new AWS.SES();
+const uuid = require('uuid');
+require('dotenv').config();
+
+const mailformat = /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/;
```

Comment:
Extract the `mailformat` regex into a util file from where you can import it and use it wherever you need it. Also, for naming a variable use usually the camel case like `mailFormat`.